Src/Support/Google.Apis.Tests/Apis/Requests/RequestBuilderTest.cs

```diff
@@ -80,7 +92,10 @@ public void TestBasePlusPath(string baseUri, string path, string expectedRequest
 
             var request = builder.CreateRequest();
             Assert.Equal(HttpMethod.Put, request.Method);
+            // Verify both the URI and the string representation.
+            // Required because URI equality hides some special character processing.
             Assert.Equal(new Uri(expectedRequestUri), request.RequestUri);
+            Assert.Equal(new Uri(expectedRequestUri).AbsoluteUri, request.RequestUri.AbsoluteUri);
```

Comment:
Great - this is the line I was looking for :)

Reply:
Happy to oblige!